Automattic/sensei
# Combine PHP and JS strings in POT generation command

Fixes #5480 
Fixes https://github.com/Automattic/sensei/issues/5297
Fixes https://github.com/Automattic/sensei/issues/5298
Fixes https://github.com/Automattic/sensei/issues/5299
Fixes https://github.com/Automattic/sensei/issues/5300

### Changes proposed in this Pull Request

* Uses same technique to generate the POT file that Sensei Pro.
* I think this broke between 4.4.2 and 4.4.3 (probably in the `babel` [update](https://github.com/Automattic/sensei/compare/version/4.4.2...version/4.4.3#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519)).

### Testing instructions
- Run `npm run i18n:build`
- Check generated POT file for issues.

(Note: PHP warnings come up depending on your PHP version but seems fine)
